docs/release.md

```diff
@@ -0,0 +1,26 @@
+# Release Process
+This document describes how to cut a `cri` plugin release.
+
+## Step 1: Update containerd vendor
+Before cutting a release, update the containerd vendor to latest
```

Comment:
update the version of containerd located in containerd/cri/vendor.conf to the latest version of containerd for the desired branch of containerd,

Reply:
Done